src/projects/pull-request/reviewers-list.html

```diff
@@ -118,8 +118,6 @@
         </div>
       </div>
     </template>
-    <set-reviewers hidden$="[[!editable]]" pull-request="{{pullRequest}}"></set-reviewers>
```

Comment:
The error about setting reviewers originates here.
The firing of resort does not work anymore, because reviewers-list is not a parent of set-reviewers anymore.